resources/views/admin.blade.php

```diff
@@ -27,49 +27,85 @@
 
         <main class="container" >
             <h1 class="h1 text-center mt-4" >Manage Employee Profiles</h1>
-            <button class="btn btn-primary rounded-circle p-0 mt-2 mb-2 "  data-bs-toggle="modal" data-bs-target="#modal-employee"  >
+            <button id="btn-add-employee" class="btn btn-primary rounded-circle p-0 mt-2 mb-2 "  data-bs-toggle="modal" data-bs-target="#modal-employee"  >
                 <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="currentColor" class="bi bi-plus" viewBox="0 0 16 16">
                     <path d="M8 4a.5.5 0 0 1 .5.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3A.5.5 0 0 1 8 4"/>
                   </svg>
             </button>
 
-            <div class="modal fade" id="modal-employee" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
-                <div id="validation-errors-create" style="display: none;" role="alert"></div>
-                <div class="modal-dialog modal-dialog-centered">
-                  <div class="modal-content">
+            <div class="modal fade " id="modal-employee" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
```

Comment:
please do the save and edit in same model instead of repeating the codes again again. I identified 2 models different models have been used. make it as one modal

Reply:
@Raza9798 okay ill do the both actions in the same modal and update you

Reply:
I successfully implanted both edit and create functions to use a single modal @Raza9798 please review again #31